maintainers/maintainer-list.nix

```diff
@@ -18357,6 +18357,13 @@
     githubId = 1108325;
     name = "Théo Zimmermann";
   };
+  zmitchell = {
```

Comment:
Erase your evidences of fixups.
Squash the commits modifying this file into only one.